ingests/ingests_api/src/main/scala/weco/storage_service/ingests_api/responses/CreateIngest.scala

```diff
@@ -65,11 +68,37 @@ trait CreateIngest[UnpackerDestination] extends FutureDirectives with Logging {
       invalidRequest(description)
     )
 
+  private def convertToBadRequestResponse(
+    invalidArg: IllegalArgumentException): Future[Route] = {
+    def originalMessage = invalidArg.getMessage
+    def prefix = "requirement failed: External identifier"
+    def newMessage =
+      if (originalMessage.startsWith(prefix))
+        "Invalid value at .bag.info.externalIdentifier:" + originalMessage
+          .stripPrefix(prefix)
+      else originalMessage
+
+    createBadRequestResponse(newMessage)
+  }
+
   private def triggerIngestStarter(
     requestDisplayIngest: RequestDisplayIngest
   ): Future[Route] = {
 
-    val ingest = requestDisplayIngest.toIngest
+    Try {
+      requestDisplayIngest.toIngest
+    } match {
+      case Failure(exception) =>
+        exception match {
+          case invalidArg: IllegalArgumentException =>
+            convertToBadRequestResponse(invalidArg)
+          case otherException => throw otherException
+        }
+      case Success(ingest) => triggerIngestStarter(ingest)
```

Comment:
I wonder if we could collapse this into a single `case` statement:
```suggestion
      case Failure(invalidArg: IllegalArgumentException) =>
        convertToBadRequestResponse(invalidArg)
      case Failure(exception) => throw exception
      case Success(ingest) => triggerIngestStarter(ingest)
```